tools/dependency/utils.py

```diff
@@ -39,11 +39,11 @@ def get_github_release_from_urls(urls):
         if components[5] == 'archive':
             # Only support .tar.gz, .zip today. Figure out the release tag from this
             # filename.
-            if components[6].endswith('.tar.gz'):
-                github_version = components[6][:-len('.tar.gz')]
+            if components[-1].endswith('.tar.gz'):
+                github_version = components[-1][:-len('.tar.gz')]
             else:
-                assert (components[6].endswith('.zip'))
-                github_version = components[6][:-len('.zip')]
+                assert (components[-1].endswith('.zip'))
```

Comment:
cc @moderation i had to switch this to using the last part as a source package has the format `archive/refs/tag/{version}.tar.gz`

Reply:
`https://github.com/bazelbuild/rules_proto/archive/4.0.0.tar.gz` still works but I'm seeing more artifacts use teh `refs/tags` link. Will your change to `utils.py` support the old and the new URL format?

Reply:
i think so - it does the same match on (part 5) for archive, but instead of using "part 6", it uses "the last part" when checking the filename